functions/index.ts

```diff
@@ -1,12 +1,13 @@
 'use strict';
```

Comment:
Nice! Happy to see your first pull request, good job :)

Did you try it on your own firebase project?
Try it with a couple of images (both JPG and PNG) of different sizes (as big as 1024x1024, and small as 50x50), and visually look at the results (in different quality options; see my comments below).
Email me some screenshots :)